### ISSUE TITLE
ImportError from conda install

### ISSUE THREAD

**Issue by isysjo** [scikit-geometry/scikit-geometry#64]
Hello, I have a relatively simple problem:
As stated in the title, I get an ImportError when I attempt to use the package (screenshot below)
<img width="562" alt="Screen Shot 2021-03-15 at 9 57 01 AM" src="https://user-images.githubusercontent.com/30419265/111164887-d61f2d00-8574-11eb-9f18-4bc53b172af7.png">

I followed the conda installation prior to this, but (and updated conda and redid the installation), but I still seem to get this error. I suspect that it is a small error on my part, but any guidance would be appreciated.

For reference, I use a OS X system (macOS Big Sur), and I am running conda though Rosetta.

Thank you in advance!

**Comment by resonating-sirsh** [comment]
I had this issue (on M1) and i went around in many many circles re/installing gmp (which i made the mistake of installing with brew and have yet to recover.)
In the end i cloned this repo and installed it with `python -m pip install -e . -v` after installing CGAL. Although it takes a while to build, it does work - and I would probably still be trying to fix my gmp. 
But, I am also curious if anyone can point out a way to clean up the gmp bit.

Incidentally, I am now trying to install `scikit-geometry` on a Docker image with conda and its take forever...

**Comment by wolfv** [comment]
Hi both, does one of you want to submit scikit-geometry to the Conda-forge rebuild for osx-arm64? https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/osx_arm64.txt

That should fix these kinds of issues and a native version of scikit-geometry would be available.

**Comment by resonating-sirsh** [comment]
yes, i can take a look at this on the weekend or early next week
